packages/bot/src/stores/footer-extension-store.js

```diff
@@ -0,0 +1,14 @@
+import {
+    action,
+} from 'mobx';
+
+export default class FooterExtensionStore {
+    constructor(root_store) {
+        this.root_store = root_store;
+    }
+
+    @action.bound
+    setActiveTab(tab) {
```

Comment:
why dont put in the main content store?, there is no attributes in this store at all

Reply:
@mustofa-binary fair point